src/components/Disputes/panels/AppealPanel.js

```diff
@@ -112,8 +112,9 @@ const AppealPanel = React.memo(function AppealPanel({
         dispute.lastRoundId,
         appealOption.outcome
       )
-      await tx.wait()
+
       onDone()
+      await tx.wait()
     } catch (err) {
       console.log('Error submitting tx: ', err)
```

Comment:
should we use `console.error` for errors?

Reply:
Yes, i think we should. Initially this was kind of a placeholder to catch the error thinking that in the future we would be handling this in the signer panel.

Reply:
cool, so in the meanwhile we can change those `console.log` for `console.error` right?